lwr_hw/include/lwr_hw/lwr_hw.h

```diff
@@ -27,6 +27,12 @@
 #include <kdl/chaindynparam.hpp> //this to compute the gravity verctor
 #include <kdl_parser/kdl_parser.hpp>
 
+namespace hardware_interface
+{
+  /// \ref JointCommandInterface for commanding cartesian-based joints. DUMMY CLASS TO TRIGGER CARTESIAN INTERFACE
+  class PositionCartesianInterface : public JointCommandInterface {};
```

Comment:
I really don't like abusing the `JointCommandInterface` this way. Why not work on a proper cartesian impedance interface?